src/status_im2/contexts/chat/messages/list/view.cljs

```diff
@@ -229,7 +236,7 @@
            :size            :heading-1
            :style           {:margin-top (if group-chat 54 12)}
            :number-of-lines 1}
-          display-name
+          (truncate-display-name display-name)
```

Comment:
I'm not sure about truncating the display name to a fixed length of 22. Did you try changing the `name-container`'s `flex-direction` property to `column`? from my understanding, there is no need for using row there. Correct me if I'm wrong.

Reply:
@ajayesivan display-name is inside `quo/text` with `:number-of-lines 1` and it causes issue on android. 
but in case i also tried `column` in `name-container` and it didn't help

Reply:
resolved